packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -224,6 +226,12 @@ export class FilamentViewer extends LitElement {
 
     this[$updateSize]();
 
+    /*
+    this[$renderer].setClearOptions(
+        {clearColor: [r, g, b, 1], clear: true, discard: true});
+    */
+    this[$view].setClearColor([1000, 1000, 1000, 1]);
+
     // Wait two rAFs to ensure we rendered at least once:
     requestAnimationFrame(() => {
       requestAnimationFrame(() => {
```

Comment:
Does it work with just one raf? If so, let's simplify; it may also speed up rendering a bit.

Reply:
Same for the other renderers.